R/ui_elements.R

```diff
@@ -11,8 +11,6 @@ use_telemetry <- function(id = "") {
   if (id != "" && !is.null(id)) {
     shiny_namespace <- glue::glue("{id}-")
   }
```

Comment:
(optional) Another improvement from the other PR, we can sneak this here :smile: 

```suggestion
  if (!is.null(id) && !identical(trimws(id), "")) {
    shiny_namespace <- shiny::NS(trimws(id), "")
  }
```